git/bootstrap.sh

```diff
@@ -1,7 +1,7 @@
 GIT_CONFIG_PATH="$DOTFILE_INSTALL_PATH/gitconfig.symlink";
 
 git_clone() {
-    git clone $1 $2 &>> $LOGDIR/git.log;
+    git clone $1 $2 >> $LOGDIR/git.log;
```

Comment:
@bsans I'm going to leave this as-is but might commit over it later when I re-install this script setup locally.